From acd5d830990975310998d28800cad086357a80c0 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 11 Nov 2011 10:48:17 +0100
Subject: [PATCH] man: fix typo in sd_notify

Noticed by guzu.
(cherry picked from commit 9f84624270432cdff35c4f499fbdb9e0f94fe705)
---
 man/sd_notify.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/man/sd_notify.xml b/man/sd_notify.xml
index dd0ba93..62347f8 100644
--- a/man/sd_notify.xml
+++ b/man/sd_notify.xml
@@ -166,7 +166,7 @@
                 for details.</para>
 
                 <para><function>sd_notifyf()</function> is similar to
-                <function>sd_notifyf()</function> but takes a
+                <function>sd_notify()</function> but takes a
                 <function>printf()</function>-like format string plus
                 arguments.</para>
         </refsect1>
-- 
1.7.7.5

